plugins/BEdita/Core/tests/TestCase/Model/Action/SignupUserActionTest.php

```diff
@@ -397,6 +397,63 @@ public function testExceptionSendMail()
         }
     }
 
+    /**
+     * Test `Signup.activationUrl` config
+     *
+     * @return void
+     */
+    public function testActivationUrl()
+    {
+        Configure::write('Signup.activationUrl', 'https://my.activation.url');
+
+        $data = [
+            'data' => [
+                'username' => 'testsignup',
+                'password' => 'testsignup',
+                'email' => 'test.signup@example.com',
+            ],
+        ];
+
+        EventManager::instance()->on('Auth.signup', function (...$arguments) {
+            static::assertCount(4, $arguments);
```

Comment:
In case this event listener is never dispatched, the test is still green. I'd add a rough check:

```php
$invoked = 0;
EventManager::instance()->on('Auth.signup', function (...$arguments) use (&$invoked) {
    $invoked++;
    // ...
});

$action = new SignupUserAction();
$action($data);
static::assertSame(1, $invoked);
```